app/models/user.rb

```diff
@@ -456,7 +457,8 @@ def has_linked_account?(provider)
   NOTIFICATIONS_TYPES = %w(assignments_notification recent_notification
                            assignments_email_notification
                            recent_email_notification
-                           system_message_email_notification)
+                           system_message_email_notification
+                           popover_v2_toggle )
```

Comment:
I don't think we should modify `NOTIFICATIONS_TYPES`.

Reply:
if I remove it from here, it doesn work, since the user doesnt have this user.popovers_enabled attribute anymore.